docker/Dockerfile

```diff
@@ -1,35 +1,35 @@
-FROM debian:trixie-slim
+FROM alpine
+RUN \
+    apk update && apk upgrade && \
+    apk add --no-cache curl
 
-ARG DEBIAN_FRONTEND=noninteractive
-ENV DEBIAN_FRONTEND=$DEBIAN_FRONTEND
+# [SR-4632] Swift on Alpine Linux (support musl libc)
+# https://github.com/swiftlang/swift/issues/47209
+# Swift on Alpine not yet supported
+#
+# RUN curl -s https://archive.swiftlang.xyz/install.sh | bash && \
```

Comment:
When this ran, I got the error that the operating system was unsupported.

Reply:
musl support is an open issue under certain conditions, but that looks like more of an arbitrary restriction of the installer script. I need to test it in a real env.

Reply:
I get it. Swift can crosscompile to support musl and so alpine as target. It cannot run on it yet.

Reply:
Ok. For reference: [Building from source](https://swift-linux.readthedocs.io/en/latest/others.html#building-from-source) or the options above it. That could be done in the Dockerfile.